test/browser/Gemfile

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
```

Comment:
Does this exist so the browser tests can use a different MR version to the rest of the tests?

Reply:
Yes, exactly, and adding it caused some issues with the Docker image, hence the removal above - although I'm not sure that was actually necessary now given that it builds without deleting what I intended it to.